packages/cache-handler/src/helpers/get-timeout-redis-command-options.ts

```diff
@@ -3,5 +3,5 @@ import { commandOptions } from 'redis';
 type CommandOptions = ReturnType<typeof commandOptions>;
 
 export function getTimeoutRedisCommandOptions(timeoutMs: number): CommandOptions {
-    return commandOptions({ signal: AbortSignal.timeout(timeoutMs) });
+    return commandOptions(!!timeoutMs && timeoutMs !== 0 ? { signal: AbortSignal.timeout(timeoutMs) } : {});
```

Comment:
Let's rely on Typescript and JSDoc here and do not check `!!timeoutMs`. Also let's use more straightforward approach using `if` statement.

```suggestion
    if (timeoutMs === 0) {
        return commandOptions({});
    }

    return commandOptions({ signal: AbortSignal.timeout(timeoutMs) });
```